tests/test_flask.py

```diff
@@ -276,6 +282,10 @@ def test_new_project(self):
             }
 
             output = c.post(
+                '/project/new', data=false_data, follow_redirects=True)
+            self.assertTrue('Please enter a URL.' in output.data)
+
```

Comment:
When speaking about moving up, I was thinking about the whole block not just ``false_data``.